alerta/utils/config.py

```diff
@@ -1,6 +1,130 @@
 import os
-
 from flask import Flask
+from voluptuous import Schema, Url, Email
+import logging
+import ast
+
+
+LOG = logging.getLogger('alerta.config')
+
+
+class Validate:
+    def __init__(self):
+        self.string_validator = Schema(str)
+        self.integer_validator = Schema(int)
+        self.list_validator = Schema(list)
+        self.url_validator = Schema(Url())
+        self.email_validator = Schema(Email())
+
+    def get_variable(self, variable_name):
+        environment_string = os.environ[variable_name]
+
+        return environment_string
+
+    def validate_boolean(self, variable_name, boolean_string):
+        if boolean_string == 'True' or boolean_string == 'true' or boolean_string == '1':
+            variable_value = True
+            LOG.info(
+                f'Environment variable {variable_name} parsed with value {variable_value}')
+
+        elif boolean_string == 'False' or boolean_string == 'false' or boolean_string == '0':
+            variable_value = False
+            LOG.info(
+                f'Environment variable {variable_name} parsed with value {variable_value}')
```

Comment:
I realise you asked specifically about `f'` strings but I thought you meant the `"".format()` strings. I didn't even know you could do this until I saw your code. I'd prefer to have only two different ways of formatting strings, if that's ok. So if you could convert these to 'format()' or '%s' that would be great. Sorry for messing you about.

Reply:
No worries, will update. The fstring format was introduced in python 3.6. Neat way of doing string formatting :)